lib/config/presets/internal/merge-confidence.ts

```diff
@@ -0,0 +1,21 @@
+import type { Preset } from '../types';
+
+export const presets: Record<string, Preset> = {
+  default: {
+    description: 'Enable Merge Confidence badges for pull requests.',
+    packageRules: [
+      {
+        matchDatasources: ['maven', 'npm', 'pypi'],
+        matchUpdateTypes: ['patch', 'minor', 'major'],
+        prBodyColumns: [
+          'Package',
+          'Change',
+          'Age',
+          'Adoption',
+          'Passing',
+          'Confidence',
```

Comment:
I'm a little tempted to make the default Age + Confidence and then offer a second "expanded" preset which includes all four

Reply:
@JamieMagee @viceice @secustor any thoughts on that?

Reply:
sounds good to me

Reply:
should i split it then, or maybe in a separate PR ?

Reply:
should I split the preset then ?

Reply:
Yes, let's have mergeConfidence:default and mergeConfidence:extended

Reply:
Alternatively, let's keep feature parity and refer to the existing one as `full`. I'll make some suggestions